assets/sass/components/overlay-notification/_googlesitekit-overlay-notification.scss

```diff
@@ -98,4 +98,18 @@
 			padding-top: 0;
 		}
 	}
+
+	&.googlesitekit--overlay-notification--reader-revenue-manager {
```

Comment:
I would suggest moving the styles for this component to its own stylesheet, e.g. `assets/sass/components/reader-revenue-manager/_googlesitekit-rrm-overlay-notification.scss`.